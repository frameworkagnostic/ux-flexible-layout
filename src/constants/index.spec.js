
import * as properties from './index';

describe('unit constants', () => {
  test('test unit', () => expect(properties.UNIT).toEqual(expect.any(String)));
  test('test step', () => expect(properties.STEP).toEqual(expect.any(Number)));
});

describe('to level api', () => {
  test('UNIT', () => expect(properties.UNIT).toBeDefined());
  test('STEP', () => expect(properties.STEP).toBeDefined());
  test('BREAKPOINT_KEYS', () => expect(properties.BREAKPOINT_KEYS).toBeDefined());
  test('BREAKPOINTS', () => expect(properties.BREAKPOINTS).toBeDefined());
  test('GRID12_COLUMN', () => expect(properties.GRID12_SIZES).toBeDefined());
  test('GRID12_GUTTERS', () => expect(properties.GRID12_GUTTERS).toBeDefined());
  test('snapshot', () => expect(properties).toMatchSnapshot());
});
