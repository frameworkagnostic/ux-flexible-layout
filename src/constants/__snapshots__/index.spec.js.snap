// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`to level api snapshot 1`] = `
Object {
  "BREAKPOINTS": Object {
    "lg": 1280,
    "md": 960,
    "sm": 600,
    "xl": 1920,
    "xs": 0,
  },
  "BREAKPOINT_KEYS": Array [
    "xs",
    "sm",
    "md",
    "lg",
    "xl",
  ],
  "GRID12_GUTTERS": Array [
    0,
    8,
    16,
    24,
    32,
    40,
  ],
  "GRID12_SIZES": Array [
    "auto",
    true,
    1,
    2,
    3,
    4,
    5,
    6,
    7,
    8,
    9,
    10,
    11,
    12,
  ],
  "STEP": 5,
  "UNIT": "px",
}
`;
